packages/react/file-input/src/FileInput.ts

```diff
@@ -63,7 +63,7 @@ const FileInput: React.ComponentType<FileInputProps> = withApollo<FileInputProps
       let response = null;
 
       try {
-        response = await client.query({ query: FILE_UPLOAD_INFO_QUERY });
+        response = await client.query({ query: FILE_UPLOAD_INFO_QUERY, fetchPolicy: 'network-only' });
```

Comment:
move it to options, please

Reply:
Do you meen that User should be able to set fetchPolicy like this:

```
<FileInput fetchPolicy="network-only" onUploadDone={ onUploadDone } value={ value } accept="image/*" public>
      { ({ pick, value }) => <Avatar css={{ flexShrink: 0 }} src={ value ? value.downloadUrl : null } firstName={ firstName } lastName={ lastName } onPick={ () => pick() } pickVariant="fullWidth" { ...rest } /> }
    </FileInput>
```

Reply:
Done